live-examples/css-examples/fonts/font-feature-settings.css

```diff
@@ -0,0 +1,22 @@
+@font-face {
+    font-family: 'Fira Sans';
+    src: local('FiraSans-Regular'),
+         url('../../../media/fonts/FiraSans-Regular.woff2') format('woff2');
```

Comment:
Is this going to have the same problem that you fixed in https://github.com/mdn/interactive-examples/pull/661?

Reply:
It certainly would. Fixed!